### DISCUSSION TITLE
importing a file doesn't work on mobile

### DISCUSSION THREAD

**Discussion by BladeNothing** [ppy/osu#27216] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Whenever I've tried to import a skin, beatmap, or images and files, it opens the file instead of selecting the file.

### Screenshots or videos

https://github.com/ppy/osu/assets/147674482/f466522e-6451-4f25-973b-2ff14403cedb

### Version

2024.131.0

### Logs

I don't know how to submit logs on mobile :P

**Comment by frenzibyte** [comment]
Can you confirm you don't actually have a folder by that name in your storage? It's likely that you're witnessing https://github.com/ppy/osu/issues/11911.

**Reply by BladeNothing** [reply]
![Screenshot_20240217-185213~2](https://github.com/ppy/osu/assets/147674482/ba2c0828-39e9-4a0a-84fb-4a832d4c43ef)

**Reply by frenzibyte** [reply, marked as answer]
Looks like a folder to me.

**Reply by BladeNothing** [reply]
Does that mean I have to turn it into a zip?

**Reply by frenzibyte** [reply]
> Does that mean I have to turn it into a zip?

Any osu! file such as a beatmap/skin/score should be a normal file suffixed with the correct extension, not a folder, nor a zip file.
